libs/blocks/quiz-entry/quizoption.js

```diff
@@ -89,13 +89,6 @@ export const GetQuizOption = ({
     }
   };
 
-  useEffect(() => {
```

Comment:
We don't need this?

Reply:
Feature wasn't fully developed, shouldn't be there yet.